src/lib/image/bind/bind.c

```diff
@@ -99,10 +99,7 @@ int _singularity_image_bind(struct image_object *image) {
 #endif
 
     singularity_message(DEBUG, "Calculating image offset\n");
-    if ( ( lo64.lo_offset = singularity_image_offset(image) ) < 0 ) {
-        singularity_message(ERROR, "Could not obtain message offset of image\n");
-        ABORT(255);
-    }
+    lo64.lo_offset = singularity_image_offset(image);
```

Comment:
`lo_offset` is unsigned and `singularity_image_offset` never returns anything less then 0.